Update details in 05-setting-permissions.md about raw querys (#14921)
metabase/metabase: docs/administration-guide/05-setting-permissions.md

#### SQL (or native query) access

- **Write raw queries:** can write new SQL/native queries using the SQL editor. This access level requires the group to additionally have Unrestricted data access for the database in question, since SQL queries can circumvent table-level permissions.

> - **No access**: can't view, write or edit SQL/native queries using the SQL editor. Users with this level of access will still be able to view the results of questions created from SQL/native queries, but not the SQL code itself. Users with this level of permissions won't see the button "View the SQL" when doing "Custom Questions".
> 

Reviewer: ```suggestion
- **No access**: can't view, write, or edit SQL/native queries. Users will still be able to view the results of questions created from SQL/native queries, but not the code itself. They also won't see the "View the SQL" button when composing custom questions in the notebook editor.

```
